source/guides/administrator.rst

```diff
@@ -14,9 +16,11 @@ Getting Started
    /getting-started/welcome_email.rst
 
 
-Installing Mattermost
+Install Mattermost
```

Comment:
If we change these titles, then we'll need to update them elsewhere in the documentation if they are used as a reference in other docs.